t/22-dashboard.t

```diff
@@ -100,8 +100,20 @@ like(
 );
 
 $t->get_ok('/dashboard_build_results')->status_is(200);
-@h2 = $t->tx->res->dom->find('h2 a')->map('text')->each;
-is_deeply(\@h2, ['opensuse test', 'Test parent'], 'parent group shown and opensuse is no more on top-level');
+@h2 = $t->tx->res->dom->find('h2 a')->map(sub ($e) { $e->text . ($e->attr('title') // '') })->each;
+my $test_overview_tooltip = 'Shows the latest test results for all job groups within this parent job group';
+is_deeply(
+    \@h2,
+    ['opensuse test', 'Test parent', $test_overview_tooltip],
+    'parent group shown and opensuse is no more on top-level'
+);
+$t->element_exists('#test_result_overview_link_1', 'The parent group should has the link to tests/overview');
```

Comment:
"should has" is wrong, either rephrase it like
```suggestion
$t->element_exists('#test_result_overview_link_1', 'parent group has link to tests/overview');
```
or remove the check altogether because the subsequent check will fail anyways (as the method `attr` would be called on `undef`).

Reply:
Done